openapi_python_client/parser/properties/__init__.py

```diff
@@ -302,6 +306,9 @@ def build_model_property(
         name=name,
         additional_properties=additional_properties,
     )
+    if prop.reference.class_name in schemas.models:
+        raise NameClashException(f'Attempted to generate duplicate models with name "{prop.reference.class_name}"')
```

Comment:
We should return a `PropertyError` here instead so our higher level error handling code can manage it rather than printing a stack trace.

Reply:
👍. Thanks for the review!